entrypoints/content/components/secondary/comment/CommentAction.tsx

```diff
@@ -0,0 +1,71 @@
+
+import { useState } from 'react'
+
+
+// Components::
+import { ArrowBigDown, ArrowBigUp, Ellipsis, Forward, MessageSquare } from 'lucide-react'
+
+// Types:
+import { VoteCount } from 'types/votes'
+
+interface VoteCountProps {
+  voteCount: VoteCount
+}
+
+const CommentAction: React.FC<VoteCountProps> = ({ voteCount: { up = 0, down = 0 } }) => {
+
+  const [voteCount, setVotecount] = useState(up - down)
+  const [isUpVoteActive, setUpVoteActive] = useState(true)
+  const [isDownVoteActive, setDownVoteActive] = useState(false)
+
+  useEffect(() => {
+    setVotecount(up - down)
+  }, [up, down])
+
+  // #TODO: Logic is incorrect need to fix this
+  // Maybe take all the states into one single object state
+  const upVote = () => {
```

Comment:
would be better if the name of the function were more specific, like `upvoteComment`

same for `downvoteComment`